api/include/vehicles/kia_niro.h

```diff
@@ -0,0 +1,460 @@
+/**
+ * @file kia_niro.h
+ * @brief Kia Niro specific macros.
+ *
+ */
+
+
+#ifndef _KIA_NIRO_PLATFORM_INFO_H_
```

Comment:
In the `can_protocols/*.h` files we use the `pack` pragma, but in the `vehicles/*.h` files we do not. This isn't a blocker for this PR but we should file a ticket to deal with the struct packing.